server/services/store/sqlstore/migrations/migrations_files/000015_blocks_history_no_nulls.up.sql

```diff
@@ -0,0 +1,85 @@
+-- parent_id
+UPDATE {{.prefix}}blocks_history bh1
+	SET parent_id = COALESCE(
+		(SELECT bh2.parent_id 
+		FROM {{.prefix}}blocks_history bh2
+		WHERE bh1.id = bh2.id AND bh2.parent_id IS NOT NULL 
+		ORDER BY bh2.insert_at ASC limit 1) 
+	, '')
+WHERE parent_id IS NULL;
+
+-- schema
+UPDATE {{.prefix}}blocks_history bh1
+	SET schema = COALESCE(
+		(SELECT bh2.schema 
+		FROM {{.prefix}}blocks_history bh2
+		WHERE bh1.id = bh2.id AND bh2.schema IS NOT NULL 
+		ORDER BY bh2.insert_at ASC limit 1) 
+	, 1)
+WHERE schema IS NULL;
+
+-- type
+UPDATE {{.prefix}}blocks_history bh1
+	SET type = COALESCE(
+		(SELECT bh2.type 
+		FROM {{.prefix}}blocks_history bh2
+		WHERE bh1.id = bh2.id AND bh2.type IS NOT NULL 
+		ORDER BY bh2.insert_at ASC limit 1) 
+	, '')
+WHERE type IS NULL;
+
+-- title
+UPDATE {{.prefix}}blocks_history bh1
+	SET title = COALESCE(
+		(SELECT bh2.title 
+		FROM {{.prefix}}blocks_history bh2
+		WHERE bh1.id = bh2.id AND bh2.title IS NOT NULL 
+		ORDER BY bh2.insert_at ASC limit 1) 
+	, '')
+WHERE title IS NULL;
+
+-- fields
+{{if .postgres}}
+	UPDATE {{.prefix}}blocks_history bh1
+		SET fields = COALESCE(
+			(SELECT bh2.fields 
+			FROM {{.prefix}}blocks_history bh2
+			WHERE bh1.id = bh2.id AND bh2.fields IS NOT NULL 
+			ORDER BY bh2.insert_at ASC limit 1) 
+		, '{}'::json)
+	WHERE fields IS NULL;
+{{else}}
+	UPDATE {{.prefix}}blocks_history bh1
+		SET fields = COALESCE(
+			(SELECT bh2.fields 
+			FROM {{.prefix}}blocks_history bh2
+			WHERE bh1.id = bh2.id AND bh2.fields IS NOT NULL 
+			ORDER BY bh2.insert_at ASC limit 1) 
+		, '')
+	WHERE fields IS NULL;
+{{end}}
+
+-- create_at
+UPDATE {{.prefix}}blocks_history bh1
+	SET create_at = update_at
```

Comment:
Accordingly to the previous code, the create at is the create at of the original block, that I think is not necesarilly the create at of the history entry. I would guess this needs a migration similar to the previous one, or we need to change the code to make the `createAt`, of the history entry the date of the creation of the history entry, not the creation of the original block.

Reply:
I agree.  I'll modify to get the correct createAt.

Reply:
Done